### PR TITLE
Add randomizer for speakers

### PR DESCRIPTION
**Pull request by beelarr**
@kwelch Is this what you had in mind?

### CODE REVIEW

**Comment by beelarr** [comment]
Ok, hold up.  Locally it was working, but in the preview the pics aren't following the rest of the content. I'll have to look into it tomorrow.

File: src/gatsby-theme-conference/components/speakers.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { Styled } from 'theme-ui'
+import { Box, Container } from 'theme-ui/layout'
+import SectionHeading from '../../../node_modules/gatsby-theme-conference/src/components/section-heading'
```
Comment (kwelch) [inline src/gatsby-theme-conference/components/speakers.js]: You can reference this as you would other node modules with out the relative path

File: src/gatsby-theme-conference/components/speakers.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    py={5}>
+    <Container>
+      <SectionHeading>
+        Speakers
+      </SectionHeading>
+      <FlexList
+        css={{
+          justifyContent: 'center',
+        }}>
+        {speakers.sort(() => 0.5 - Math.random())
```
Comment (kwelch) [inline src/gatsby-theme-conference/components/speakers.js]: This should work. What if we keep the keynotes at the top and slice them then randomize the rest. 

I realize I am just adding more and more complexity 😁

**Review by kwelch — APPROVED** [review]

File: src/data/speakers.yml [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 - id: karaluton
   name: Kara Luton
   image: https://pbs.twimg.com/profile_images/1010566462782701568/6sDbHiB0_400x400.jpg
   twitter: https://twitter.com/karaluton
   github: https://github.com/karaluton
-  bio: " "
-  company: " "
+  bio: ' '
+  company: ' '
+- id: unknown
```
Comment (kwelch) [inline src/data/speakers.yml:22]: Removing this may actually be fine I saw a version where it looked good.

**Comment by itzsaga** [comment]
@kwelch I don't have insight into this. However I'm closing it due to staleness. Can revisit if needed.

**Comment by kwelch** [comment]
This was just so that the speakers would not always show in the same order making it same as though we had ranked them.

**Pull request closed without merging**